Resources/translations/FOSUserBundle.en.yml

```diff
@@ -50,9 +50,12 @@ registration:
 
 resetting:
     password_already_requested: 'The password for this user has already been requested within the last 24 hours.'
-    check_email: 'An email has been sent to %email%. It contains a link you must click to reset your password.'
+    check_email: |
+        An email has been sent. It contains a link you must click to reset your password.
+        Note: You can only request a new password within %tokenLifetime% hours.
+
+        If you don't get an email check your spam folder or try again.
```

Comment:
You can use this for german please!

```
Eine E-Mail wurde verschickt. Sie beinhaltet einen Link zum Zurücksetzen des Passwortes.
Hinweis: Du kannst ein neues Passwort nur alle %tokenLifetime% Stunden beantragen.

Bitte überprüfe deinen Spam-Ordner, falls du die E-Mail nicht erhalten hast.
```

@core23 are you fine with my translation?